### DISCUSSION TITLE
椭圆滤波器 | boringhex.top 博客

### DISCUSSION THREAD

**Discussion by giscus** [boringhex-top/blog.boringhex.top#34] [Announcements]
# 椭圆滤波器 | boringhex.top 博客

之前的文章 信号去噪 中列出了7种常用的信号去噪算法，对于后两种算法——深度学习和奇异值分解(SVD)，我现在也不太理解，就先不写了。 很多朋友留言又提了一些算法，今天一起来聊聊椭圆滤波器。 椭圆滤波器（Elliptic Filter），也称为Cauer滤波器，是一种数字滤波器，用于信号处理和滤波应用。椭圆滤波器之所以得名，是因为其在频率响应图上的特征呈现出椭圆形的形状。 椭圆滤波器有以下特点和优

https://blog.boringhex.top/posts/b477d4bc10fa/

**Comment by Bduoluoluo** [comment]
nihao

**Reply by pengwon** [reply]
你好呀😀

**Reply by no010** [reply]
这个是回复谁呢
